.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
     env:
       USE_NINJA: "1"
       CC: "clang" # used by the wasm2c tests
-      WASM2C_CFLAGS: "-fsanitize=address"
+      WASM2C_CFLAGS: "-fsanitize=address -fno-sanitize-recover"
```

Comment:
Wow.. under what circumstances do ASAN failures don't cause the program to return non-zero?   I had no idea this extra flag was needed.

Reply:
Looking at the docs it looks like only UBSan has the `no-sanitize-recover` flag and that it designed to be used *instead* of `-fsantitize=`.  So it would be `-fno-sanitize-recover=undefined` ?

Reply:
Just took the `-fno-sanitize-recover` off of ASAN. I was really just trying to copy the behavior we already have in CMakeLists.txt, where it appends `-fno-sanitize-recover` for the UBSAN run. (And you're right, only for UBSAN.) Then after #2081 we start automatically propagating the exact flags from CMakeLists.txt into WASM2C_CFLAGS (including `-fno-sanitize-recover`) so we can stop hardcoding it in build.yml.

Reply:
Just made it `-fno-sanitize-recover=undefined` per request. It will become automatic with #2081 anyway (will match the exact flags from CMakeLists.txt).